p/themes/Alternative-Dark/template.css

```diff
@@ -868,7 +868,21 @@ br {
 	display: block;
 }
 
-/*=== "Load more" part */
+/*=== "Load" parts */
+#stream {
+	position: relative;
+	z-index: 5;
+}
+
+#first_load {
+	min-height: 40px;
+	min-width: 40px;
+	position: fixed;
+	top: 200px;
+	width: calc(100% - 300px);
```

Comment:
Note to self:
https://developer.mozilla.org/en-US/docs/Web/CSS/calc#Browser_compatibility